test/blackbox-tests/test-cases/melange/simple.t/dune-project

```diff
@@ -2,4 +2,4 @@
 
 (using melange 0.1)
 
-(package (name x))
+(package (allow_empty) (name x))
```

Comment:
why do we need this package here? the library `x` is private anyway.

Reply:
not needed, removed in https://github.com/ocaml/dune/pull/6349/commits/52569d962d026b70b58d9fa99820af977cfc513b.